setup.py

```diff
@@ -45,7 +45,6 @@
     include_package_data=True,
     keywords='iam_profile_faker',
     name='iam_profile_faker',
-    packages=find_packages(include=['iam_profile_faker']),
```

Comment:
Why are you removing this? It is the package that it should install by default. I guess this breaks the cli since you are adding the actual package only to ``api_requirements``.